### PR TITLE
feat: add FullStory and a cookie consent banner

### PR DESCRIPTION
**Pull request by skyrpex**
Closes #5 and #6.

### CODE REVIEW

**Comment by skyrpex** [comment]
@eladb the cookie consent currently looks like this:

image

I'm also tracking the `commandExecuted` event in FullStory.

File: src/components/CookieConsent.vue [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+  return true;
+});
+</script>
+
+<!-- This example requires Tailwind CSS v2.0+ -->
+<template>
+  <TransitionRoot as="template" :show="open">
+    <Dialog as="div" class="relative z-10 pointer-events-none">
+      <!-- <TransitionChild
```
Comment (eladb) [inline src/components/CookieConsent.vue:26]: Delete?

File: src/analytics.ts [hunk 2]
```diff
@@ -1,13 +1,20 @@
...
+const AMPLITUDE_API_KEY = "0dd9d988f0137cbb9d3bd337a00fe257";
+const FULLSTORY_ORG_ID = "o-1CTP1V-na1";
+
+export async function setupAnalytics() {
+  FullStory.init({ orgId: FULLSTORY_ORG_ID });
+  await amplitude.init(AMPLITUDE_API_KEY).promise;
+}
+
+export function trackCommandExecuted(command: string) {
+  amplitude.track("commandExecuted", {
```
Comment (eladb) [inline src/analytics.ts:13]: move "commandExecuted" to a const

File: src/components/CookieConsent.vue (hunk 1 above)
Comment (skyrpex) [reply src/components/CookieConsent.vue:26]: That element (if shown) can block the console until the consent is accepted. Maybe we want to do that.

File: src/components/CookieConsent.vue [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+                      class="text-lg leading-6 font-medium text-gray-900"
+                    >
+                      Cookie consent
+                    </DialogTitle>
+                    <div class="mt-2">
+                      <p class="text-sm text-gray-500">
+                        We use cookies to improve user experience and analyze
+                        website traffic. By clicking “Accept”, you consent to
+                        store Analytics and Customization Cookies on your
+                        device.
```
Comment (eladb) [inline src/components/CookieConsent.vue:70]:
```suggestion
                        We use cookies to improve user experience and analyze
                        website traffic. By clicking "Accept", you consent to
                        store analytics cookies on your device.
```

File: src/components/CookieConsent.vue (hunk 1 above)
Comment (eladb) [reply src/components/CookieConsent.vue:26]: Let's start without it. I think it's fine. (I understand this technically means that we are storing a cookie before users provided their consent)

Comment (skyrpex) [reply src/components/CookieConsent.vue:26]: Otherwise, I'll just remove the comment.

Comment (skyrpex) [reply src/components/CookieConsent.vue]: > Let's start without it. I think it's fine. (I understand this technically means that we are storing a cookie before users provided their consent)

Yeah :) It happens a lot, though!

**Review by eladb — APPROVED** [review]

**Pull request merged**